pkg/kubelet/apis/device-plugin/v1alpha1/constants.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deviceplugin
+
+const (
+	// Healthy means that the device is healty
+	Healty = "Healthy"
+	// UnHealthy means that the device is unhealty
+	Unhealthy = "Unhealthy"
+
+	// HeartbeatOk means that the heartbeat succeeded
+	HeartbeatOk = "Heartbeat ok"
+	// HeartbeatFailure means that the heartbeat did not succeed
+	HeartbeatFailure = "Heartbeat Failure"
+
+	// Version is the API version
+	Version = "alpha"
+	// DevicePluginPath is the folder the Device Plugin is expecting sockets to be on
+	DevicePluginPath = "/var/run/kubernetes/"
+	// KubeletSocket is the path the Kubelet registry socket
+	KubeletSocket = DevicePluginPath + "kubelet.sock"
+
```

Comment:
Can we leave the rest of the const string definitions to later PRs where they are used? That would provide better context on how they will be used.